nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-properties-loader/src/test/groovy/org/apache/nifi/properties/ProtectedNiFiPropertiesGroovyTest.groovy

```diff
@@ -662,7 +630,7 @@ class ProtectedNiFiPropertiesGroovyTest extends GroovyTestCase {
     @Test
     void testInstanceWithNoProtectedPropertiesShouldNotLoadSPP() throws Exception {
         // Arrange
-        ProtectedNiFiProperties properties = loadFromFile("/conf/nifi.properties")
+        ProtectedNiFiProperties properties = loadFromFile("/conf/nifi.properties", KEY_HEX)
         assert properties.@localProviderCache?.isEmpty()
```

Comment:
I think `localProviderCache` can be removed.

Reply:
Good catch; removed.